pkg/clients/clients.go

```diff
@@ -654,6 +654,8 @@ func GetTestClients(tcp TestClientParams) *Settings {
 
 	if len(tcp.K8sMockObjects) > 0 && len(tcp.SchemeAttachers) > 0 {
 		genericClientObjects = append(genericClientObjects, tcp.K8sMockObjects...)
+	} else {
+		clientSet.Interface = dynamicFake.NewSimpleDynamicClient(clientSet.scheme, genericClientObjects...)
```

Comment:
Why does it need to be moved?

Reply:
otherwise it panic on operatorGroup struct. Anyway we are plan to remove dynamic client soo. After this PR got merged: https://github.com/openshift-kni/eco-goinfra/pull/536